### PR TITLE
new style guide

### PR DESCRIPTION
**Pull request by Balendra-Maharjan**
Things Done:
- [x] Styleguide Typography

TODO:
- [form styling ] 
- [header]
- [footer]
- [page naavigation]

### CODE REVIEW

File: dev/index.html [hunk 1]
```diff
@@ -1,28 +1,120 @@
...
-    <head>
-        @@include('includes/head.html',{'pageTitle':'FED Starter Kit'})
-    </head>
-    <body>
-        <div class="theme-app d-flex flex-column">
-            <div class="theme-wrap position-relative">
-                @@include('includes/header.html', {
-                    'headerSkin': 'theme-header--dark'
+<head>
+    @@include('includes/head.html',{'pageTitle':'FED Starter Kit'})
```
Comment (synsuman) [inline dev/index.html:4]: change title name

File: dev/scss/layout/_layouts.scss [hunk 2]
```diff
@@ -0,0 +1,30 @@
+.content-with-image-a01 {
+    h2 {
+        font-size: 3.5rem;
```
Comment (synsuman) [inline dev/scss/layout/_layouts.scss:3]: why do we need this?

File: dev/scss/layout/_layouts.scss [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    }
+    &__icon {
+        height: 68px;
```
Comment (synsuman) [inline dev/scss/layout/_layouts.scss:6]: what is this file for?

File: dev/scss/styles/leadspace.scss [hunk 4]
```diff
@@ -1 +1,7 @@
 @import '../common-imports';
+
+.leadspace__img-group {
+    @include media-breakpoint-up(lg) {
+        font-size: 2.8rem;
```
Comment (synsuman) [inline dev/scss/styles/leadspace.scss:5]: why are we using font size here?

File: dev/scss/_variables.scss [hunk 5]
```diff
@@ -603,7 +607,10 @@ $aspect-ratios: (
 "1x1": 100%,
 "4x3": calc(3 / 4 * 100%),
 "16x9": calc(9 / 16 * 100%),
-"21x9": calc(9 / 21 * 100%)
+"21x9": calc(9 / 21 * 100%),
+"327x233": calc(233 / 327 * 100%),
+"376x376": calc(376 / 376 * 100%),
```
Comment (synsuman) [inline dev/scss/_variables.scss:612]: 1*1 will work for this

File: dev/scss/_variables.scss [hunk 6]
```diff
@@ -603,7 +607,10 @@ $aspect-ratios: (
...
+"327x281": calc(281 / 327 * 100%)
```
Comment (synsuman) [inline dev/scss/_variables.scss:613]: are these all default needed for project?

File: dev/index.html (hunk 1 above)
Comment (Balendra-Maharjan) [reply dev/index.html:4]: done

**Pull request merged**